pkg/apis/addtoscheme_kubevirt_v1alpha1.go

```diff
@@ -1,7 +1,7 @@
 package apis
 
 import (
-	"kubevirt.io/web-ui-operator/pkg/apis/kubevirt/v1alpha1"
+	"github.com/kubevirt/web-ui-operator/pkg/apis/kubevirt/v1alpha1"
```

Comment:
The `kubevirt.io` was selected in the pre-HCO era to be same with the `kubevirt`.
As more operators are implemented now, I agree with moving the project under `github.com` but this needs to be done systematically for all imports. I will push a more complex PR for that.